src/organisms/navigation/tab/index.js

```diff
@@ -19,6 +20,7 @@ const Tab = ({
   dragHandleProps,
   tabRef,
   showBorderLeft,
+  tooltip = "",
```

Comment:
Why is this initialization needed?

Reply:
To avoid undefined errors, on the other hand haven't test  it without it

Reply:
Actually I think that if no `tooltip` content is provided by the user, then it should not be visible, that's why I made the question! 😄